deploy/operator/no_ironic/operator.yaml

```diff
@@ -12,7 +12,7 @@ spec:
       labels:
         name: metal3-baremetal-operator
     spec:
-      serviceAccountName: metal3-baremetal-operator
+      serviceAccountName: capm3-metal3-baremetal-operator
```

Comment:
don't change this. it breaks the backwards compatibility. you need to change it through kustomize the same way you did for the clusterrolebinding